src/MOL_discretization.jl

```diff
@@ -1,22 +1,80 @@
+# Method of lines discretization scheme
 struct MOLFiniteDifference{T} <: DiffEqBase.AbstractDiscretization
   dxs::T
   order::Int
 end
 MOLFiniteDifference(args...;order=2) = MOLFiniteDifference(args,order)
 
+# Evaluate expression
+function eval_expr(expr,x,y)
+   f = eval(:((x,y) -> $expr))
+   return @eval $f.($x,$y)
+end
+
+# Obtain boundary conditions from an array
+function extract_bc(bcs,tdomain,domain)
+    u_t0 = 0.0
+    u_x0 = 0.0
+    u_x1 = 0.0
+    n = size(bcs)[1]
+    for i = 1:n

```

Comment:
I don't know why is there a loop if we only return scalars.

Reply:
We return scalars or expressions (see u_t0 = Expr(bcs[i].rhs). Also, this function will increase its complexity by adding Neumann/Robin BCs and 2D/3D BCs.

We need the loop because the BCs can be presented in any order, e.g.:
```
bcs = [u(0,x) ~ -x*(x-1)*sin(x),
       u(t,0) ~ 0.0,
       u(t,1) ~ 0.0]
```
or
```
bcs = [ u(t,0) ~ 0.0,
        u(t,1) ~ 0.0,
        u(0,x) ~ -x*(x-1)*sin(x)]
```
etc.